src/main/app/src/app/admin/params/params.component.less

```diff
@@ -0,0 +1,4 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
```

Comment:
Ik had onlangs een discussie met Bart of deze lege files nodig waren. Van mij mogen ze weg maar volgens Bart niet. Ik liet de beslissing verder aan Bart over. Ik weet niet wat Bart uiteindelijk besloten heeft